interface/skin.dmf

```diff
@@ -366,8 +368,8 @@ window "infowindow"
 		anchor2 = -1,-1
 		saved-params = "pos;size;is-minimized;is-maximized"
 		is-pane = true
-	elem "info"
-		type = INFO
+	elem "statbrowser"
+		type = BROWSER
```

Comment:
``winset: Element infowindow.info not found.``
Seems the TGUI theme stuff wasn't updated with it